src/main/java/com/gmail/nossr50/skills/taming/TamingManager.java

```diff
@@ -287,6 +287,9 @@ public void attackTarget(LivingEntity target) {
         double range = 5;
         Player player = getPlayer();
 
+        if (!target.getWorld().equals(player.getWorld()))
```

Comment:
When would a player's world not equal their targets?

Reply:
It's possible to shoot an arrow through a nether portal and hit an entity on the other side of it

Reply:
Thank you for that information!